app/src/main/kotlin/com/softteco/template/ui/feature/signUp/SignUpScreen.kt

```diff
@@ -173,29 +184,36 @@ private fun PasswordField(
 ) {
     Column {
         var passwordVisibility by remember { mutableStateOf(true) }
-        OutlinedTextField(value = state.passwordValue, onValueChange = {
-            state.onPasswordChanged(it)
-        }, modifier = modifier, label = {
-            Text(text = stringResource(id = R.string.password))
-        }, isError = state.fieldStatePassword is PasswordFieldState.Empty, trailingIcon = {
-            IconButton(onClick = {
-                passwordVisibility = !passwordVisibility
-            }) {
-                Icon(
-                    imageVector = if (passwordVisibility) {
-                        Icons.Default.Create
-                    } else {
-                        Icons.Default.Done
-                    },
-                    contentDescription = stringResource(id = R.string.visibility),
-                    tint = MaterialTheme.colorScheme.primary
-                )
+        OutlinedTextField(
+            value = state.passwordValue,
+            onValueChange = {
+                state.onPasswordChanged(it)
+            },
+            modifier = modifier,
+            label = {
+                Text(text = stringResource(id = R.string.password))
+            },
+            isError = state.fieldStatePassword is PasswordFieldState.Empty,
+            trailingIcon = {
+                IconButton(onClick = {
+                    passwordVisibility = !passwordVisibility
+                }) {
+                    Icon(
+                        imageVector = if (passwordVisibility) {
+                            Icons.Default.Create
+                        } else {
+                            Icons.Default.Done
+                        },
+                        contentDescription = stringResource(id = R.string.visibility),
```

Comment:
R.string.visibility - the description does not correspond to the logical purpose of the field